cpp/src/slice2java/msbuild/slice2java.vcxproj

```diff
@@ -68,7 +68,6 @@
       <AdditionalIncludeDirectories>..;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
       <PreprocessorDefinitions>ICE_BUILDING_SLICE_COMPILERS;%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <RuntimeLibrary>MultiThreadedDebug</RuntimeLibrary>
-
```

Comment:
I noticed these random newlines in `slice2java` when I was cross-checking the files in `slice2swift` to the other compilers. The other ones I checked didn't have these newlines here, and they're in a strange place to begin with.
So I removed them.